### PR TITLE
upgrade to rabbitmq 3.7

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

**Comment by lazedo** [comment]
the changes for amqp_util -> kz_amqp_util are necessary because rabbitmq client has an amqp_util module as of 3.7

File: Makefile [hunk 1]
```diff
@@ -202,7 +202,7 @@ bump-copyright:
 $(FMT):
 	wget -qO - 'https://codeload.github.com/fenollp/erlang-formatter/tar.gz/master' | tar xz -C $(ROOT)/make/
 
-fmt: TO_FMT ?= $(shell git --no-pager diff --name-only HEAD origin/master -- "*.erl" "*.hrl" "*.escript")
+fmt: TO_FMT ?= $(shell git --no-pager diff --name-only HEAD origin/master | grep -e  "\.erl\|\.hrl\|\.escript")
```
Comment (jamesaimonetti) [inline Makefile]: Curious why this change was made?

Comment (lazedo) [reply Makefile]: seems related to my git version on my old CentOS 6 dev box.
i'll remove this
